extensions/theme-defaults/themes/light_modern.json

```diff
@@ -131,7 +131,7 @@
 		"textCodeBlock.background": "#F8F8F8",
 		"textLink.activeForeground": "#005FB8",
 		"textLink.foreground": "#005FB8",
-		"textPreformat.foreground": "#3B3B3B",
+		"textPreformat.foreground": "#A31515",
```

Comment:
This is required for `code` blocks in Markdown previews in Light Modern to render in a distinct color